posts/2021-05-07-caught-red-handed.md

```diff
@@ -1,11 +1,36 @@
 ---
 layout: post
-title: "1.52.0, fingerprints and compiler error forensics"
-author: Felix Klock
+title: "Rust 1.52.1"
+author: Felix Klock, Mark Rousskov
 team: the compiler team <https://www.rust-lang.org/governance/teams/compiler>
+release: true
 ---
 
-The Rust teams are always excited to report on new features offered with each release. Sometimes, however, an important change that is not yet "fully baked" gets accidentally included in a release.
+The Rust team has prepared a new release, 1.52.1, working around a bug
+introduced in 1.52.0. We recommend all Rust users, including those currently
```

Comment:
From what I understand of previous comments yinz have made, the bug(s) weren't *introduced* in 1.52.0 so much as *surfaced* or *made into a compiler error*. I'd also mention incremental compilation in the first sentence. So perhaps something like:

> The Rust team has prepared a new release, 1.52.1, working around a bug in incremental compilation that was made into a compiler error in 1.52.0.